test/fixtures/sinon-stub/output.js

```diff
@@ -0,0 +1,36 @@
+describe("sinon-stub", () => {
+  let testContext;
+  const obj = {
+    method: (num) => num + 10,
+    add: (a, b) => a + b,
+  };
+
+  function setupMock(ss) {
+    this.stubs = {
+      add: jest.spyOn(obj, "add").mockImplementation(() => {
+        $.Deferred().resolve();
+      }),
```

Comment:
Don't forget to return the deferred!
```suggestion
      add: jest.spyOn(obj, "add").mockImplementation(() => $.Deferred().resolve()),
```

Reply:
Making it return statement so that if it returns a complex object, we can still handle